viz-lib/src/visualizations/map/getOptions.ts

```diff
@@ -1,6 +1,31 @@
 import { merge } from "lodash";
 
-const DEFAULT_OPTIONS = {
+export type LeafletBaseIconType = "marker" | "rectangle" | "circle" | "rectangle-dot" | "circle-dot" | "doughnut";
+export interface MapOptionsType {
+  latColName: string;
+  lonColName: string;
+  classify: any;
+  groups: object;
```

Comment:
please use `Record<string, any>` rather than `object`. if you know the structure of a group it's better to use actual structure than `any`.